### PR TITLE
updated README

### PR DESCRIPTION
**Pull request by ArtiVlasov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,140 @@ Rules
...
+                    true,
+                    true
+                ]
+            ]
+        },
+        "value_type": "bool"
+    }
+```
+
+the value translations are a list of lists for your desired style. Each individual value translation is a list of two elements: `[ursgal_param_value, tool_param_value]`.
```
Comment (fu) [inline README.md:101]: should read `[uparma_param_value, tool_param_value]`

File: README.md [hunk 2]
```diff
@@ -24,6 +24,140 @@ Rules
...
+**Note:** to make the whole translation logic work in any direction, the value translations **MUST** also be provided for `ursgal_style_1`. As you can see in the example, the value is simply mapped to itself.
```
Comment (fu) [inline README.md:102]: not needed, I think.

File: README.md [hunk 3]
```diff
@@ -24,6 +24,140 @@ Rules
...
+**Note:** to make the whole translation logic work in any direction, the value translations **MUST** also be provided for `ursgal_style_1`. As you can see in the example, the value is simply mapped to itself. 
+
+
+### **Special cases**
+
+Some key_translations values cannot be directly mapped to a tool specific parameter. This is due to the harmonization of parameters in this project and different parameter handling of the tools. For example, if considereing tolerance parameters some tools accept symmetric windows, while others can handle asymmetric windows. The representation of such can be either one parameter, where the tool internally applies it in both direction from the measured value (only for symmetric windows), or in turn a list of lower and upper boundary. 
+
+These parameters require further tool specific processing, prior being injected e.g. into a command line. You can easily identify them, as they have a **\<tag\>** in their names. In the following example:
+```
+"cutadapt_style_1": "-u_<part_read_start>",
```
Comment (fu) [inline README.md:111]: is 1 missing in part1?

File: README.md [hunk 4]
```diff
@@ -24,6 +24,140 @@ Rules
...
+"cutadapt_style_1": "-u_<part2_read_end>",
+```
+
+the tool cutadapt expects two values following the **-u** parameter. For matter of harmonization, as other tools might expect it in a different format, these two parameters are separated and flagged as `<part1>` and `<part2>`. During command line preparation for cutadapt, these parameters will be caught and formatted to the expected format.
+
+A special case bulid the parameters ending with a **\<DROP_KEY\>** tag.
```
Comment (fu) [inline README.md:117]: A special case are parameters ending with a **\<DROP_KEY\>** key_translation

**Review by fu — CHANGES_REQUESTED** [review]
Awesome Artyom! Thanks!!!! I added two / three comments of things I think you have missed :)

File: README.md (hunk 3 above)
Comment (ArtiVlasov) [reply README.md:111]: @tristan-ranff will it break something in cut adapt if I update the param name? Not sure if you are calling the full tag in your wrapper

File: README.md (hunk 2 above)
Comment (ArtiVlasov) [reply README.md:102]: the whole block or just the NOTE?

**Pull request merged**